api/desecapi/models.py

```diff
@@ -99,7 +99,7 @@ def _limit_domains_default():
     is_active = models.BooleanField(default=True)
     is_admin = models.BooleanField(default=False)
     created = models.DateTimeField(auto_now_add=True)
-    limit_domains = models.IntegerField(default=_limit_domains_default.__func__, null=True, blank=True)
+    limit_domains = models.PositiveIntegerField(default=_limit_domains_default.__func__, null=True, blank=True)
```

Comment:
Does this include zero? We may want to set `limit_domains = 0` one day.

Reply:
yes, it does (https://docs.djangoproject.com/en/3.2/ref/models/fields/#positiveintegerfield)

Reply:
Weird name for a field that allows zero. :nerd_face: